lib/std/os.zig

```diff
@@ -217,9 +217,13 @@ pub const socket_t = if (builtin.os.tag == .windows) windows.ws2_32.SOCKET else
 pub var environ: [][*:0]u8 = undefined;
 
 /// Populated by startup code before main().
-/// Not available on Windows. See `std.process.args`
-/// for obtaining the process arguments.
-pub var argv: [][*:0]u8 = undefined;
+/// Not available on WASI without libc, or on Windows. See `std.process.argsAlloc`
+/// or `std.process.argsWithAllocator` for obtaining the process arguments.
+pub var argv: [][*:0]u8 = switch (builtin.os.tag) {
+    .windows => @compileError("argv isn't supported on Windows: use std.process.argsAlloc instead"),
```

Comment:
Looking at this again, Windows should also have `argv` if linking libc.

Reply:
Thanks for pointing this out :)